index.js

```diff
@@ -1,18 +1,36 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color ="pink"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
+ console.log('This is what the traveler object looks like: ', traveler)
+let first = document.querySelector('[id="profile"] h2')
+first.textContent= `${traveler.name}`
+let nickname= document.querySelector('[id="profile"] em')
+nickname.textContent = `${traveler.nickname}`
+let pic =  document.querySelector('[id="profile"] img')
+pic.src = `${traveler.photo}`
+pic.alt =`${traveler.name}`
```

Comment:
Good! One suggestion I have is that there is no need for [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) since you're not string interpolating


```suggestion
let first = document.querySelector('[id="profile"] h2')
first.textContent= traveler.name

let nickname= document.querySelector('[id="profile"] em')
nickname.textContent = traveler.nickname

let pic =  document.querySelector('[id="profile"] img')
pic.src = traveler.photo
pic.alt = traveler.name
```